dist-git.spec

```diff
@@ -85,70 +86,37 @@ cd -
 # /etc/ .......... config files
 # ------------------------------------------------------------------------------
 install -d %{buildroot}%{_sysconfdir}/dist-git
 install -d %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git
-install -d %{buildroot}%{_sysconfdir}/cron.d/dist-git
 mkdir -p   %{buildroot}%{_unitdir}
 
-cp -a configs/dist-git/dist-git.conf  %{buildroot}%{_sysconfdir}/dist-git/
-cp -a configs/gitolite/gitolite.rc    %{buildroot}%{_sysconfdir}/dist-git/
-cp -a configs/httpd/dist-git.conf     %{buildroot}%{_sysconfdir}/httpd/conf.d/
-cp -a configs/httpd/ssl.conf.example  %{buildroot}%{_sysconfdir}/httpd/conf.d/
+cp -a configs/httpd/dist-git.conf %{buildroot}%{_sysconfdir}/httpd/conf.d/
 cp -a configs/httpd/dist-git/* %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git/
-cp -a configs/cron/*           %{buildroot}%{_sysconfdir}/cron.d/dist-git/
 cp -a configs/systemd/*        %{buildroot}%{_unitdir}/
 
 # ------------------------------------------------------------------------------
 # /var/lib/ ...... dynamic persistent files
 # ------------------------------------------------------------------------------
-install -d %{buildroot}%{_sharedstatedir}/dist-git
-install -d %{buildroot}%{_sharedstatedir}/dist-git/git
-install -d %{buildroot}%{_sharedstatedir}/dist-git/git/rpms
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite/conf
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite/logs
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite/local
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite/local/VREF
-install -d %{buildroot}%{_sharedstatedir}/dist-git/cache
-install -d %{buildroot}%{_sharedstatedir}/dist-git/cache/lookaside
-install -d %{buildroot}%{_sharedstatedir}/dist-git/cache/lookaside/pkgs
-install -d %{buildroot}%{_sharedstatedir}/dist-git/web
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite/hooks
-install -d %{buildroot}%{_sharedstatedir}/dist-git/gitolite/hooks/common
-
-cp -a scripts/httpd/upload.cgi %{buildroot}%{_sharedstatedir}/dist-git/web/
-
-cp -a scripts/git/hooks/update-block-push-origin \
-      %{buildroot}%{_sharedstatedir}/dist-git/gitolite/local/VREF/update-block-push-origin
-
-ln -f -s %{_sysconfdir}/dist-git/gitolite.rc \
-         %{buildroot}%{_sharedstatedir}/dist-git/git/.gitolite.rc
-
-ln -f -s %{_sharedstatedir}/dist-git/gitolite \
-         %{buildroot}%{_sharedstatedir}/dist-git/git/.gitolite
-
-ln -f -s %{_sharedstatedir}/dist-git/git/rpms \
-         %{buildroot}%{_sharedstatedir}/dist-git/git/repositories
+install -d %{buildroot}%{installdir}
+install -d %{buildroot}%{installdir}/git
+install -d %{buildroot}%{installdir}/git/repositories
+install -d %{buildroot}%{installdir}/cache
+install -d %{buildroot}%{installdir}/cache/lookaside
+install -d %{buildroot}%{installdir}/cache/lookaside/pkgs
+install -d %{buildroot}%{installdir}/web
```

Comment:
Turns out I was lost in the review :( when I reviewed patch-by-patch.  This is relevant though:
https://github.com/release-engineering/dist-git/pull/10#discussion_r101022639